h/accounts/models.py

```diff
@@ -167,6 +167,10 @@ def _hash_password(self, password):
         return text_type(CRYPT.encode(password + self.salt))
 
     @classmethod
+    def get_by_id(cls, id_):
+        return cls.query.filter(cls.id == id_).first()
```

Comment:
I'm not really convinced we need a `get_by_id` class method. But even if we do, it should do `return cls.query.get(id_)`.